system/src/system_network_compat.cpp

```diff
@@ -255,6 +255,86 @@ class NetworkAccessTechnologyDiagnosticData: public AbstractIntegerDiagnosticDat
     }
 };
 
+class NetworkCellularCellGlobalIdentityMobileCountryCodeDiagnosticData
+    : public AbstractIntegerDiagnosticData
+{
+public:
+    NetworkCellularCellGlobalIdentityMobileCountryCodeDiagnosticData()
+        : AbstractIntegerDiagnosticData(
+              DIAG_ID_NETWORK_CELLULAR_CELL_GLOBAL_IDENTITY_MOBILE_COUNTRY_CODE,
+              DIAG_NAME_NETWORK_CELLULAR_CELL_GLOBAL_IDENTITY_MOBILE_COUNTRY_CODE)
+    {
+    }
+
+    virtual int get(IntType& val)
+    {
+        const int some_value = 310;
```

Comment:
I'm guessing these 4 get() methods are WIP

Reply:
Yes, using them to get the framing out there. They will be changed.